src/main/java/address/util/DragContainer.java

```diff
@@ -0,0 +1,32 @@
+package address.util;
+
+import javafx.scene.input.DataFormat;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ *To contain a list of Integer.
+ */
+public class DragContainer implements Serializable{
```

Comment:
Why does this need to be serialized, and are there advantages this class has over a standard `List<Integer>`?